web3.md

```diff
@@ -1588,7 +1588,7 @@ Collecting Coverage Data
  // ------------------------------------------------------------------
     rule #makeCoverageReport (.List                                         , _       , _   ) => .JSONs
     rule #makeCoverageReport ((ListItem({ CODEHASH | EPHASE } #as KEY) KEYS), COVERAGE, PGMS) => {
-                                                                                                  "hash": Int2String(CODEHASH),
+                                                                                                  "bytecode": CODEHASH,
```

Comment:
Let's rename the variable `CODEHASH` to `BYTECODE` to better reflect what it is.